src/actions/groups.ts

```diff
@@ -40,8 +36,6 @@ export function linkGroupSyncable(groupID: string, syncableID: string, syncableT
             console.warn(`unhandled syncable type ${syncableType}`); // eslint-disable-line no-console
         }
 
-        dispatches.push({type: GroupTypes.LINK_GROUP_SYNCABLE_SUCCESS, data: null}, {type, data});
```

Comment:
Aah the test case was failing because of removing second part of the code here. This should be 

`dispatches.push({type, data});`

Reply:
Good catch. I was stuck on this for a while